models.py

```diff
@@ -1401,3 +1404,40 @@ def __init__(self, *args, **kwargs):
 			nonlinearity = ('leaky_relu', 0.01),
 			**kwargs
 		)
+
+
+class OnnxWrapper():
```

Comment:
why are you not deriving from `nn.Module`? were there problem with this?

Reply:
Now nested from nn.Module